security/tpm/package.json

```diff
@@ -55,10 +55,10 @@
       ".tsx"
     ],
     "check-coverage": true,
-    "lines": 70,
+    "lines": 68,
     "functions": 77,
-    "branches": 55,
-    "statements": 71
+    "branches": 54,
+    "statements": 69
```

Comment:
Interesting that changing the target reduced our test coverage

Reply:
Probably because there's more (or different) boilerplate code in the generated module that deals with downlevel versions.